src/common/strutil.cpp

```diff
@@ -48,7 +48,7 @@ std::string toUpper(std::string str) {
 }
 
 bool contains(const std::string &str, const std::string &s) {
-	return std::regex_match(str, std::regex(".*" + s + ".*"));
+	return str.find(s) != std::string::npos;
```

Comment:
I'll try that after everything else is resolved, but it may screw up commit history in the branch, making either merge process or final commit history uglier.

Reply:
Update: I've managed to separate it from original commit, but that operation has changed commit hashes, making pulls/pushes a lot harder. The simplest solution would be to reopen the pull request by making a new branch with this change separated.